### ISSUE TITLE
Global "use strict" false positive

### ISSUE THREAD

**Issue by NN---** [palantir/tslint#344]
TSLint says "use strict" is missing in the following code:

``` typescript
"use strict";

module A { // Complains here
}
```

**Comment by leeavital** [comment]
The "use strict" goes inside the module block.

````typescript
module A {
    "use strict";
}
````

This behaviour is consistent with check-function.

```typescript
// fails
"use strict";
function foo() {}

// passes
function foo() {
 "use strict";
}

**Comment by adidahiya** [comment]
well, we should probably still find a way to make this work if users don't want to use internal `module`s (which are now called `namespace`s)

**Comment by ajafff** [comment]
`use-strict` was removed in TSLint 4.0.0. That makes this issue obsolete
